unsupported/juno/crates/pass/src/passes/strip_flow.rs

```diff
@@ -0,0 +1,424 @@
+                builder.type_parameters(None);
+                return node.replace_with_new(
+                    builder::Builder::ClassDeclaration(builder),
+                    gc,
+                    self,
+                );
+            }
+            Node::ClassExpression(n) => {
+                let mut builder = builder::ClassExpression::from_node(n);
+                builder.implements(vec![]);
+                builder.super_type_parameters(None);
+                builder.type_parameters(None);
+                return node.replace_with_new(builder::Builder::ClassExpression(builder), gc, self);
+            }
+
+            Node::ClassProperty(n) => {
+                if n.value.is_none() {
+                    return TransformResult::Removed;
+                }
+
+                let mut builder = builder::ClassProperty::from_node(n);
+                builder.type_annotation(None);
+                builder.variance(None);
+                return node.replace_with_new(builder::Builder::ClassProperty(builder), gc, self);
+            }
+            Node::ClassPrivateProperty(n) => {
+                if n.value.is_none() {
+                    return TransformResult::Removed;
+                }
+
+                let mut builder = builder::ClassPrivateProperty::from_node(n);
+                builder.type_annotation(None);
+                builder.variance(None);
+                return node.replace_with_new(
+                    builder::Builder::ClassPrivateProperty(builder),
+                    gc,
+                    self,
+                );
+            }
+
+            Node::CallExpression(n) => {
+                let mut builder = builder::CallExpression::from_node(n);
+                builder.type_arguments(None);
+                return node.replace_with_new(builder::Builder::CallExpression(builder), gc, self);
+            }
+            Node::NewExpression(n) => {
+                let mut builder = builder::NewExpression::from_node(n);
+                builder.type_arguments(None);
+                return node.replace_with_new(builder::Builder::NewExpression(builder), gc, self);
+            }
+
+            Node::EnumDeclaration(n) => {
+                return node.replace_with_existing(transform_enum(gc, n), gc, self);
+            }
+            Node::Program(n) => {
+                let mut builder = builder::Program::from_node(n);
+                let mut body = n.body.clone();
+                for i in (0..body.len()).rev() {
+                    if let Some(Node::ExportDefaultDeclaration(ExportDefaultDeclaration {
+                        metadata,
+                        declaration: enum_node @ Node::EnumDeclaration(e),
+                    })) = body.get(i)
+                    {
+                        body[i] = enum_node;
+                        body.insert(
+                            i + 1,
+                            builder::ExportDefaultDeclaration::build_template(
```

Comment:
This isn't ideal but I couldn't find something like `node.replace_with_multiple` to turn
```js
export default enum X {};
```
into
```js
enum X {};
export default X;
```

Reply:
That's a good use case, thanks. We'll try and come up with a good way to handle replace with multiple like this.